models/eval/inspirational_generation.py

```diff
@@ -136,7 +137,7 @@ def gradientDescentOnInput(model,
 
     if nevergrad not in [None, 'CMA', 'DE', 'PSO',
                          'TwoPointsDE', 'PortfolioDiscreteOnePlusOne',
-                         'DiscreteOnePlusOne', 'OnePlusOne']:
+                         'DiscreteOnePlusOne', 'OnePlusOne', 'random', 'loss-covering', 'hypervolume', 'domain-covering']:
```

Comment:
this mixes algorithms and selection mode, I dont get it and the docstring is not up to date.
Also, other modes dont seem to be used anyway since it loops on all selection options

Reply:
fixed, now there is an additional "moo", in which case we loop over various Pareto sampling modes.